Readme.md

```diff
@@ -1,33 +1,24 @@
 <!-- default file list -->
 *Files to look at*:
 
-* [Form1.cs](./CS/LocalizableDisplayName/Form1.cs) (VB: [Form1.vb](./VB/LocalizableDisplayName/Form1.vb))
+* [LocalizableDisplayNameAttribute.cs](./CS/LocalizableDisplayName/LocalizableDisplayNameAttribute.cs) (VB: [LocalizableDisplayNameAttribute.vb](./VB/LocalizableDisplayName/LocalizableDisplayNameAttribute.vb))
 * [Product.cs](./CS/LocalizableDisplayName/Product.cs) (VB: [Product.vb](./VB/LocalizableDisplayName/Product.vb))
-* [Program.cs](./CS/LocalizableDisplayName/Program.cs) (VB: [Program.vb](./VB/LocalizableDisplayName/Program.vb))
-* [PropertyNamesRes.de.resx](./CS/LocalizableDisplayName/PropertyNamesRes.de.resx) (VB: [PropertyNamesRes.de.resx](./VB/LocalizableDisplayName/PropertyNamesRes.de.resx))
-* [PropertyNamesRes.resx](./CS/LocalizableDisplayName/PropertyNamesRes.resx) (VB: [PropertyNamesRes.resx](./VB/LocalizableDisplayName/PropertyNamesRes.resx))
 <!-- default file list end -->
-# How to localize displayable names of persistent properties
+# How to localize the displayable name of a property
 
+.NET Framework has a built-in mechanism to customize a displayable name. Refer to the [DisplayNameAttribute](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.displaynameattribute) description for details.
 
-<p>This example demonstrates how to apply DXDisplayName to persistent properties. It's also required to add neutral and localized resource files to the project and set the DXDisplayNameAttribute.UseResourceManager static property to True:</p>
-
-```cs
-<newline/>
-System.ComponentModel.DXDisplayNameAttribute.UseResourceManager = true;<newline/>
-
-```
+You can create a DisplayNameAttribute descendant and override its [DisplayName](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.displaynameattribute.displayname) property. In the DisplayName property getter, retrieve a corresponding value from a resource file. Refer to the following MSDN article for additional information: [Work with .resx files programmatically](https://docs.microsoft.com/en-us/dotnet/framework/resources/working-with-resx-files-programmatically).
 
+In this example we demonstrate how to retrieve values from resource files using classes generated by the ResXFileCodeGenerator tool. You can find the source code in the following files: [LocalizableDisplayNameAttribute.cs](./CS/LocalizableDisplayName/LocalizableDisplayNameAttribute.cs) and [LocalizableDisplayNameAttribute.vb](./VB/LocalizableDisplayName/LocalizableDisplayNameAttribute.vb).
 
+To use this solution, add the LocalizableDisplayNameAttribute.cs or LocalizableDisplayNameAttribute.vb file to your project. In the Solution Explorer window, set focus on an appropriate *.resx file, press F4 to open the Properties window, and ensure that the **Custom Tool** property value is **ResXFileCodeGenerator**.
```

Comment:
- `set focus on` Explain
- `an appropriate *.resx file` Define 'appropriate'

Reply:
I changed this sentence to: "In the Solution Explorer window, **click a *.resx file that contains property names**, press F4 to open the Properties window, and ensure that the **Custom Tool** property value is **ResXFileCodeGenerator**."

Reply:
Ok